### ISSUE TITLE
debug.zip: `hot-ranges.sh` no longer works with `ranges.json`

### ISSUE THREAD

**Issue by erikgrinaker** [cockroachdb/cockroach#108164]
The `hot-ranges.sh` script wasn't updated when we merged range info into a single per-node `ranges.json` in #104133. It is not sufficient to simply update the path to use `./nodes/*/ranges.json`, since it will no longer output the range IDs that were previously embedded in the filename.

Instead, the script must be rewritten to use e.g. `jq` to merge range info from all the `ranges.json` files, sort them, and output the ranges with the highest values in a human-readable format.

Jira issue: CRDB-30340

gz#22710

**Comment by kvoli** [comment]
This is now broken on every supported release ([v22.2](https://github.com/cockroachdb/cockroach/pull/110050), [v23.1](https://github.com/cockroachdb/cockroach/pull/104248)) and [master](https://github.com/cockroachdb/cockroach/pull/104133). 

I think we should 
- Prioritize a fix, backporting to v23.1+v22.2
- Add testing since this is relied upon during investigations

**Comment by kvoli** [comment]
I took a first pass at creating a JQ query, but didn't get around to merging the jsons, so this query runs per `ranges.json`.

```
find . -path './nodes/*/ranges.json' -print0 | xargs -0 cat | jq -r '[.[] | { range_id: .state.state.desc.range_id, stats: .stats }] | sort_by(.stats.queries_per_second) | reverse | .[:10]'
```
